packages/progress-indicator/_mixins.scss

```
@@ -62,14 +62,20 @@
     left: adapter.get-spacing-size($level: xxs);
     display: block;
     box-sizing: border-box;
-    border: adapter.get-progress-indicator-border-size() solid adapter.get-progress-indicator-indicator-surface-color();
+    border-style: solid;
+    border-color: adapter.get-progress-indicator-indicator-surface-color();
     border-radius: 50%;
     transform: rotate(45deg);
     transform-origin: center;
     transition: clip-path 0.1s ease;
     content: "";
     clip-path: polygon(0% 0%, 0% 100%, 100% 100%, 100% 0%, 0% 0%, 50% 50%);
```

線の太さの初期値が存在しないので、設定をなにもしていない場合 (`-size-{size}` modifier が設定されていないとき) に s, m, l のどれでもない線の太さが適用されていそうです。デフォルト値をl にするなら、このへんに以下のようにデフォルトの線の太さを入れておくと安全そうです。


```suggestion
    clip-path: polygon(0% 0%, 0% 100%, 100% 100%, 100% 0%, 0% 0%, 50% 50%);
    border-width: adapter.get-progress-indicator-border-size(l);
```

---

https://github.com/pepabo/inhouse-components-web/pull/40/commits/9af688a33ee06549d6f31567678d383cee28cb18

こちらで対応しました